src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -71,7 +71,8 @@
     public $incrementing = true;
 
     /**
-     * Sets the type used by the ID.
+     * Sets the type used by the ID. Note that this property is ignored unless
```

Comment:
Actually, just make the description this: `The type used by the incrementing ID.`. It's pretty clear then.